src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -13,6 +16,12 @@ public class StandardOutputVisitor : XmlTestExecutionVisitor
         readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
         readonly string defaultDirectory;
         readonly bool showProgress;
+        readonly Stopwatch clock;
+        private ConcurrentDictionary<string, long> runningTests;
+        private Thread watcher;
+        readonly int longTestMaxMilliseconds = 1_000 * 60 * 5;
```

Comment:
This change appears to be breaking BuildTools. Looks like the compiler doesn't like this line. 

https://devdiv.visualstudio.com/DevDiv/Net%20CoreFX/_build/index?buildId=1577238&_a=summary

```
2018-04-10T23:28:56.6798516Z Visitors\StandardOutputVisitor.cs(22,49): error CS1002: ; expected [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
2018-04-10T23:28:56.6799210Z Visitors\StandardOutputVisitor.cs(22,56): error CS1519: Invalid token '60' in class, struct, or interface member declaration [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
2018-04-10T23:28:56.6799797Z Visitors\StandardOutputVisitor.cs(23,51): error CS1002: ; expected [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
2018-04-10T23:28:56.6800539Z Visitors\StandardOutputVisitor.cs(23,58): error CS1519: Invalid token '60' in class, struct, or interface member declaration [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
2018-04-10T23:28:56.6801176Z Visitors\StandardOutputVisitor.cs(147,76): error CS1525: Invalid expression term 'long' [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
2018-04-10T23:28:56.6801755Z Visitors\StandardOutputVisitor.cs(147,81): error CS1003: Syntax error, ',' expected [E:\A\_work\3365\s\src\xunit.console.netcore\xunit.console.netcore.csproj]
```

I suspect it has to do with `_`, do you know if that is a new compiler feature, as I've never seen constants using that.

Reply:
can the compiler be updated on official build machine? c#7 is over one year old now :)

Reply:
@dotnet/dnceng can we look into getting build tools on a machine pool that supports VS2017?

Reply:
@dotnet/dnceng @MattGal for RS4 runner work we definitely need a machine pool with VS2017 and Windows 10 RS4 SDK 17134 installed. Is that feasible?